### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by nishanthmahasamudram**

### CODE REVIEW

File: server/routes/changeTeam/handlers/changeTeam.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default class ChangeTeamRoutes {
     const probationTeamsWithCount = probationTeams.map(probationTeam => {
       return {
         ...probationTeam,
-        count: teams.find(t => t.teamCode === probationTeam.code)?.count || 0,
+        count: req.route.path.includes('create') ? 0 : teams.find(t => t.teamCode === probationTeam.code)?.count || 0,
```
Comment (nishanthmahasamudram) [inline server/routes/changeTeam/handlers/changeTeam.ts:35]: Display count only in the vary tile.

Comment (nishanthmahasamudram) [reply server/routes/changeTeam/handlers/changeTeam.ts:35]: image
image

File: server/routes/changeTeam/handlers/changeTeam.ts [hunk 2]
```diff
@@ -2,7 +2,12 @@ import { RequestHandler, Request, Response } from 'express'
 import CaseloadService from '../../../services/caseloadService'
 
 export default class ChangeTeamRoutes {
-  constructor(private readonly caseloadService: CaseloadService) {}
+  constructor(
+    private readonly caseloadService: CaseloadService,
+    private section: string
```
Comment (andrewrlee) [inline server/routes/changeTeam/handlers/changeTeam.ts:7]: little trick - you can do `private section: string` and you don't need to assign the variable in a constructor

you can refer to it as  this.section regardless

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**